docs/config_search.md

```diff
@@ -131,13 +133,12 @@ run_config_search_disable: True
 profile_models:
   model_1:
     model_config_parameters:
-        max_batch_size: [6, 8]
-        dynamic_batching:
-            max_queue_delay_microseconds: [200, 300]
-        instance_group:
-        -
-            kind: KIND_GPU
-            count: [1, 2]
+      max_batch_size: [6, 8]
+      dynamic_batching:
+        max_queue_delay_microseconds: [200, 300]
+      instance_group:
```

Comment:
is this equivalent/correct?

Reply:
This is the result of YAPF formatting. I don't know if it is equivalent, but it is now correct (this is the formatting I see in yaml files I use when running live tests).